src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionXpathSingleFilterTest.java

```diff
@@ -327,6 +327,49 @@ public void testThrowException() throws Exception {
         }
     }
 
+    @Test
+    public void testAllNullConfiguration() throws Exception {
+        final String[] expected = {
+            "18:1: " + getCheckMessage(MissingJavadocTypeCheck.class, MSG_JAVADOC_MISSING),
+        };
+
+        final String[] suppressed = CommonUtil.EMPTY_STRING_ARRAY;
+
+        verifyFilterWithInlineConfigParser(
+                getPath("InputSuppressionXpathSingleFilterAllNullConfiguration.java"),
+                expected, removeSuppressed(expected, suppressed));
+    }
+
+    @Test
+    public void testNullFirstThree() throws Exception {
```

Comment:
`testDefaultAllPropertiesExceptForIdAndExpression` put this update in Input name also

Reply:
done

Reply:
``testDefaultAllPropertiesExceptForIdAndExpression`` is too long, and I renamed it ``testDecideByIdAndExpression``.